containers/reverse-proxy/create-envoy-config.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+
+PORT0=$1
+PORT1=$2
```

Comment:
We should get rid of these and just assume SERVICE_PORT and PORT0 are in the environment.

Reply:
Done